extensions/extensions-docker/aissemble-policy-decision-point/src/main/resources/docker/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN mkdir -p /deployments/krausening/
 COPY target/dockerbuild/*.jar /deployments/
 
 COPY ./src/main/resources/truststore/aissemble-secure.jks /deployments/
-COPY ./src/main/resources/krausening/base/aissemble-security.properties /deployments/krausening/
+COPY ./src/main/resources/krausening/base/aiops-security.properties /deployments/krausening/
```

Comment:
I: This rename is the primary fix for PDP.  Because the code that reads this file hasn't been updated to look for the new name yet, it was failing to find this config.